src/main/scala/com/github/xiaodongw/swagger/finatra/WebjarsController.scala

```diff
@@ -17,17 +17,19 @@ object WebjarsController {
   private val DEFAULT_EXPIRE_TIME_MS: Long = 86400000L // 1 day
 }
 
-class WebjarsController @Inject() (resolver: FileResolver) extends Controller {
+class WebjarsController (jar: String, version: String) extends Controller {
   import WebjarsController._
 
-  private val root: String = "/webjars"
+  @Inject
+  var resolver: FileResolver = _
+
+  private val root: String = "webjars"
```

Comment:
This is inconsistent, this root is without "/" prefix, but others are with.
I prefer to have "/" prefix for all of them.